### PR TITLE
Fix doc issues related to new gepetto-viewer API modifications

### PR DESCRIPTION
**Pull request by jcarpent**
Related to #720

### CODE REVIEW

File: doc/d-practical-exercises/src/display.py [hunk 1]
```diff
@@ -50,6 +50,6 @@ def place(self,objName,M,refresh=True):
         only at the end of the list.
         '''
         self.viewer.gui.applyConfiguration(objName,
-                                           se3ToXYZQUAT(M))
+                                           pin.pin.oXYZQUATtuple(M))
```
Comment (gabrielebndn) [inline doc/d-practical-exercises/src/display.py]: I think this is a mistake

File: doc/d-practical-exercises/src/mobilerobot.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import pinocchio as se3
+import pinocchio as pin
 from pinocchio.robot_wrapper import RobotWrapper
-from pinocchio.utils import eye, se3ToXYZQUAT
+from pinocchio.utils import eye, pin.oXYZQUATtuple
```
Comment (gabrielebndn) [inline doc/d-practical-exercises/src/mobilerobot.py]: Also here

File: doc/d-practical-exercises/src/mobilerobot.py [hunk 3]
```diff
@@ -58,13 +58,13 @@ def initDisplay(self, loadModel):
     def display(self, q):
         RobotWrapper.display(self, q)
         M1 = self.data.oMi[1]
-        self.viewer.gui.applyConfiguration('world/mobilebasis', se3ToXYZQUAT(M1))
-        self.viewer.gui.applyConfiguration('world/mobilewheel1', se3ToXYZQUAT(M1))
-        self.viewer.gui.applyConfiguration('world/mobilewheel2', se3ToXYZQUAT(M1))
+        self.viewer.gui.applyConfiguration('world/mobilebasis', pin.oXYZQUATtuple(M1))
```
Comment (gabrielebndn) [inline doc/d-practical-exercises/src/mobilerobot.py]: And here (the same error repeats below)

File: bindings/python/pinocchio/utils.py [hunk 4]
```diff
@@ -27,12 +27,6 @@ def skew(p):
     x, y, z = p
     return np.matrix([[0, -z, y], [z, 0, -x], [-y, x, 0]], np.double)
 
-
-se3ToXYZQUAT = pin.se3ToXYZQUAT
```
Comment (jmirabel) [inline bindings/python/pinocchio/utils.py]: I added them in #720 for backward compatibility. Shouldn't we keep them ?

By the way,  `se3ToXYZQUAT = pin.se3ToXYZQUAT` should have been  `se3ToXYZQUAT = pin.se3ToXYZQUATtuple`

Comment (gabrielebndn) [reply bindings/python/pinocchio/utils.py]: I think they should be kept as deprecated

Comment (jcarpent) [reply bindings/python/pinocchio/utils.py]: They are already exposed in the main scope, but we can indeed retrieve the previous behavior

Comment (jcarpent) [reply bindings/python/pinocchio/utils.py]: with deprecation.

Comment (jmirabel) [reply bindings/python/pinocchio/utils.py]: > They are already exposed in the main scope

Yes, but in many examples/tutorials, I see
```python
from pinocchio.utils import *
import pinocchio as pin
```
In this case, they become unavailable.

A more concise declaration would be
```python
from pinocchio import se3ToXYZQUATtuple as se3ToXYZQUAT, XYZQUATToSe3
```
but then, they wouldn't be deprecated.

**Pull request merged**